### DISCUSSION TITLE
全网寻找出币方——联合挖矿

### DISCUSSION THREAD

**Discussion by ElvinChanAA** [filecoin-project/community-china#224] [新手上路]
香港及新加坡IDC机房，安全保障
联合方案可详谈，wechat：249841730

**Comment by ElvinChanAA** [comment]
TG:elvin0104

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
